Makefile

```diff
@@ -46,7 +46,7 @@ node_modules: build
 
 .PHONY: clean
 clean:
-	git clean -ffdxe .vscode
+	git clean -ffdxe .vscode -e compiler/src/lnntoamm
```

Comment:
what is this and why?

Reply:
ah when I was first setting up and figuring things out the dir was getting deleted because of `git clean`, so this was to prevent the dir from getting wiped - I just forgot to delete this